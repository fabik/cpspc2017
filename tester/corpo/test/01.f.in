100
45 1 69 28 44 12 93 56 14 13 46 54 18 33 65 3 35 21 7 50 64 61 95 34 2 17 31 97 94 30 91 63 8 11 40 51 74 99 92 10 20 83 78 25 87 43 77 67 66 47 80 37 72 9 16 82 76 15 96 22 29 98 58 19 86 48 26 79 6 23 53 57 85 24 5 75 100 52 70 49 90 62 32 4 38 68 89 39 27 55 41 81 60 84 42 73 71 36 88
68 1 69 58 80 42 49 56 14 72 20 11 35 59 65 3 54 86 7 50 64 61 95 34 87 17 76 97 31 94 91 63 81 18 40 83 85 99 92 10 46 51 78 25 2 37 24 67 19 47 44 43 13 9 16 90 30 33 96 22 29 71 28 66 21 75 26 79 52 98 53 57 4 77 5 48 100 6 70 93 27 62 32 74 38 45 89 39 82 55 41 8 60 84 12 73 23 36 88
