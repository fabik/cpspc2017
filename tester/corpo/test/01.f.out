99 24 98 57 11 60 20 31 45 75 79 19 79 76 0 47 97 77 49 19 49 8 6 39 29 25 96 10 90 14 14 42 2 30 77 71 84 54 69 72 74 19 84 64 28 23 85 36 34 50 84 60 83 79 44 46 93 10 0 17 7 89 3 9 48 49 35 24 58 61 4 82 92 55 36 10 36 65 59 60 31 94 88 41 55 53 24 67 68 94 43 73 31 16 40 18 91 4 70 66 
